lib/ui/pages/views/facilities_view/facilities_view.dart

```diff
@@ -88,205 +83,75 @@ class _HospitalHeaderCard extends ViewModelWidget<FacilitiesViewModel> {
-          children: <Widget>[
-            Container(
-              width: SizeConfig.screenWidth / 3,
-              child: AutoSizeText(title,
-                  maxLines: 1,
-                  style: TextStyle(
-                      fontSize: MediaQuery.of(context).size.width * 0.05,
-                      fontWeight: FontWeight.w300)),
-            ),
-            _BuilderMeter(occupied: occupied, vacant: vacant),
-          ],
-        ),
-      ),
-    );
-  }
-}
-
-class _BuilderMeter extends ViewModelWidget<FacilitiesViewModel> {
-  final int occupied, vacant;
-
-  const _BuilderMeter({Key key, @required this.occupied, @required this.vacant})
-      : super(key: key);
-  @override
-  Widget build(BuildContext context, FacilitiesViewModel model) {
-    return LayoutBuilder(
-      builder: (context, constaints) => Row(
-        children: [
-          AutoSizeText(
-            occupied?.toString() ?? '...',
-            maxLines: 1,
-            style: TextStyle(
-                fontSize: MediaQuery.of(context).size.width * 0.05,
-                color: Colors.red,
-                fontWeight: FontWeight.w300),
-          ),
-          Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 4.0),
-            child: Container(
-              height: constaints.maxHeight,
-              width: MediaQuery.of(context).size.width * 0.25,
-              decoration: BoxDecoration(
-                gradient: LinearGradient(
-                  stops: [
-                    model.calculateOccupancyRate(occupied, vacant),
-                    model.calculateOccupancyRate(occupied, vacant)
-                  ],
-                  begin: Alignment.topLeft,
-                  end: Alignment.topRight,
-                  colors: <Color>[
-                    const Color(0xffEB5757),
-                    const Color(0xff27AE60)
-                  ],
+  Widget build(BuildContext context, FutureFacilitiesViewModel model) {
+    return model.isBusy
+        ? Container()
+        : SingleChildScrollView(
+            child: Column(
+              children: <Widget>[
+                _FacilitiesTitleCard(),
+                Padding(
+                  padding: const EdgeInsets.all(25.0),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      SizedBox(height: 5),
+                      // TODO: This can be improved using a loop
+                      FacilitiesTitle(
+                          title: 'Facilities(covid):', enableLegends: true),
+                      FacilitiesDetailsRow(
+                        occupancyRate: model.calculateOccupancyRate(model.hospitalSummary?.icuVacant, model.hospitalSummary?.icuOccupied),
+                        title: 'icu beds:',
+                        occupied: model.hospitalSummary?.icuOccupied,
+                        vacant: model.hospitalSummary?.icuVacant,
+                      ),
```

Comment:
In `calculateOccupancyRate` method, occupied is the parameter and the second one is vacant, that's why we need to interchange the parameters' position for the `FacilitiesDetailsRow` widget.

Reply:
Will definitely change to this.
```diff
-occupancyRate: model.calculateOccupancyRate(model.hospitalSummary?.icuVacant, model.hospitalSummary?.icuOccupied),
+occupancyRate: model.calculateOccupancyRate(model.hospitalSummary?.icuOccupied, model.hospitalSummary?.icuVacant),
```